syntax = "proto2";

package capsule;

import "capsule.proto";

message ClientGetRequest {

  optional string hash = 1;
  optional string replyAddr = 2;

}

message ClientGetResponse {

  optional bool success = 1;
  optional string hash = 2;
  optional CapsulePDU record = 3;

  optional string targetAddr = 4;

}